tests/test_virtualscreening.py

```diff
@@ -67,7 +67,10 @@ def test_vs_docking():
     assert 'vina_affinity' in mol_data
     assert 'vina_rmsd_lb' in mol_data
     assert 'vina_rmsd_ub' in mol_data
-    vina_scores = [-6.3, -6.0, -5.8, -5.8, -3.9, -3.0, -1.1]
+    if oddt.toolkit.backend == 'ob' and oddt.toolkit.__version__ < '3.0':
```

Comment:
I think this is artifact of rebasing, I have removed the first case when deprecating OB <3.0

Reply:
yes indeed